OpenRA.Mods.Common/Traits/World/EditorResourceLayer.cs

```diff
@@ -146,7 +146,7 @@ public void Render(WorldRenderer wr)
 
 			Dirty.Clear();
 
-			foreach (var uv in wr.Viewport.VisibleCells.MapCoords)
+			foreach (var uv in wr.Viewport.VisibleCellsInsideBounds.MapCoords)
```

Comment:
This (`EditorResourceLayer`) is only used inside the editor, so why not make things clearer and use `AllVisibleCells`?